project/Project.scala

```diff
@@ -28,37 +28,38 @@ import com.typesafe.sbt.site.SphinxSupport.Sphinx
 object Zipkin extends Build {
   val zipkinVersion = "1.2.0-SNAPSHOT"
 
-  val finagleVersion = "6.26.0"
-  val utilVersion = "6.25.0"
-  val scroogeVersion = "3.19.0"
-  val zookeeperVersions = Map(
+  def finagle(name: String) = "com.twitter" %% ("finagle-" + name) % "6.26.0"
```

Comment:
On alternative option is to accept  varargs, e.g.

```scala
libraryDependencies ++= finagle("core", "http", "thrift")
```

which could be implemented with something like

```scala
def finagle(name: String*) = name map { n => "com.twitter" %% ("finagle-" + n) % "6.26.0" }
```